data/mods/Magiclysm/monsters/Orcs.json

```
@@ -0,0 +1,101 @@
+[
+  {
+    "id": "mon_orc_warrior",
+    "type": "MONSTER",
+    "name": { "str": "Orc Warrior" },
+    "description": "This green brute is a orc warrior. It stares at you menacingly, club in hand.",
+    "default_faction": "orcs",
+    "bodytype": "human",
+    "species": [ "ORCS" ],
+    "volume": "60 L",
+    "weight": "70 kg",
+    "hp": 92,
+    "speed": 110,
+    "material": [ "flesh" ],
+    "symbol": "o",
+    "color": "green",
+    "aggression": 100,
+    "morale": 110,
+    "melee_skill": 5,
+    "melee_dice": 6,
+    "melee_dice_sides": 4,
+    "dodge": 10,
+    "harvest": "orc",
+    "vision_day": 13,
+    "vision_night": 5,
+    "zombify_into": "mon_zorc",
+    "death_drops": {
+      "subtype": "collection",
+      "items": [
+        { "item": "helmet_scrap", "prob": 20 },
+        { "item": "legguard_scrap", "prob": 20 },
+        { "item": "boots_scrap", "prob": 20 },
+        { "item": "armguard_scrap", "prob": 20 },
+        { "item": "cuirass_scrap", "prob": 10 },
+        { "item": "longsword", "prob": 100 }
+      ]
+    },
+    "death_function": [ "NORMAL" ],
+    "flags": [ "SEES", "HEARS", "WARM", "BASHES", "FILTHY", "PATH_AVOID_DANGER_2" ]
+  },
+  {
+    "type": "MONSTER",
+    "id": "mon_orc_archer",
+    "name": { "str": "orc archer" },
+    "description": "This green brute is a orc archer. It looks at you with a bow in hand and smiles silently as it aims at you.",
+    "copy-from": "mon_orc_warrior",
+    "melee_skill": 2,
+    "death_drops": {
+      "subtype": "collection",
+      "items": [
+        { "item": "helmet_scrap", "prob": 40 },
+        { "item": "legguard_scrap", "prob": 40 },
+        { "item": "boots_scrap", "prob": 40 },
+        { "item": "armguard_scrap", "prob": 40 },
+        { "item": "cuirass_scrap", "prob": 40 },
+        { "item": "Longbow", "prob": 95 }
```

```suggestion
        { "item": "Longbow", "prob": 100 },
        { "item": "sheath", "contents-item": "knife_combat", "prob": 50 }
```
Other knives could be substituted for knife_combat - knife_rambo, for instance.